includes/class-wc-gateway-ppec-checkout-handler.php

```diff
@@ -994,4 +998,20 @@ public function filter_wc_checkout_params( $params ) {
 
 		return $params;
 	}
+
+	/**
+	 * Compatibility method for WC >= 3.3.
+	 *
+	 * @param array  $params
+	 * @param string $handle
+	 *
+	 * @return string URL
+	 */
+	public function filter_wc_checkout_params_post33( $params, $handle ) {
+		if ( 'wc-checkout' !== $handle ) {
```

Comment:
@bor0 we don't need 2 functions.

What if we add to the first function parameters.
`( $params, $handle = '' )`

Then we have a default so there will be no warning.

Removing this function also makes it easy to remove the second filter at a later stage.